lxd/config/map.go

```diff
@@ -95,14 +96,21 @@ func (m *Map) Change(changes map[string]interface{}) (map[string]string, error)
 func (m *Map) Dump() map[string]interface{} {
 	values := map[string]interface{}{}
 
-	for name, key := range m.schema {
-		value := m.GetRaw(name)
-		if value != key.Default {
-			if key.Hidden {
-				values[name] = true
-			} else {
-				values[name] = value
+	for name, value := range m.values {
```

Comment:
I’m not completely sure this loops over the same set, but I don’t know anything about this part of the codebase. I think it’s worth going over to make 100% this loop goes over the same set. If it does, everything else looks good to me. Good Job!